build/common.props

```diff
@@ -3,19 +3,23 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <LangVersion>7.2</LangVersion>
-    <BuildNumber Condition=" '$(BuildNumber)' == '' ">1</BuildNumber>
-    <MajorMinorProductVersion>3.0</MajorMinorProductVersion>
-    <VersionPrefix>$(MajorMinorProductVersion).$(BuildNumber)</VersionPrefix>
-    <Version Condition=" '$(VersionSuffix)' != '' ">$(VersionPrefix)-$(VersionSuffix)</Version>
+    <MajorVersion>3</MajorVersion>
+    <MinorVersion>1</MinorVersion>  <!--This will be incremented by automation-->
```

Comment:
Which automation is updating this? Should we be relying on a property set by the environment instead?

Reply:
There will be a step in release pipeline that would increment this similar to rabbbitmq. There's a tracking work item for that here. 
#7312

Reply:
Added automation here https://azfunc.visualstudio.com/Azure%20Functions/_releaseDefinition?definitionId=20&_a=definition-tasks&environmentId=209

Reply:
Removed automation step as discussed